actionmailer/lib/action_mailer/base.rb

```diff
@@ -935,6 +939,15 @@ def self.supports_path?
       false
     end
 
+    def self.class_from_string_or_symbol(klass_or_string_or_symbol)
```

Comment:
This should have a `# :nodoc:` and be set as private with `private_class_method` I believe.